src/main/java/controller/BlackjackGame.java

```diff
@@ -0,0 +1,99 @@
+package controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import util.YesOrNo;
+import domain.card.CardDeck;
+import domain.result.DealerResult;
+import domain.result.MatchCalculator;
+import domain.result.MatchResult;
+import domain.result.PlayerResult;
+import domain.user.Dealer;
+import domain.user.Player;
+import domain.user.PlayerFactory;
+import domain.user.User;
+import view.InputView;
+import view.OutputView;
+
+public class BlackjackGame {
+	private final CardDeck cardDeck;
+
+	public BlackjackGame(CardDeck cardDeck) {
+		this.cardDeck = cardDeck;
+	}
+
+	public void play() {
+		final List<Player> players = PlayerFactory.create(InputView.inputNames());
+		final Dealer dealer = new Dealer();
+		drawInitialUsersCard(players, dealer);
+		drawAdditionalUsersCard(players, dealer);
+		printAllUserScore(players, dealer);
+		printGameResult(players, dealer);
+	}
+
+	private void drawInitialUsersCard(List<Player> players, Dealer dealer) {
+		drawInitialPlayersCard(players);
+		drawInitialDealerCard(dealer);
+		OutputView.printInitialResult(players, dealer);
+	}
+
+	private void drawInitialPlayersCard(List<Player> players) {
+		for (Player player : players) {
+			player.addCards(cardDeck.draw(), cardDeck.draw());
+		}
+	}
+
+	private void drawInitialDealerCard(Dealer dealer) {
+		dealer.addCards(cardDeck.draw(), cardDeck.draw());
+	}
+
+	private void drawAdditionalUsersCard(List<Player> players, Dealer dealer) {
+		if (dealer.isBlackjack()) {
+			return;
+		}
+		drawAdditionalPlayersCard(players);
+		drawAdditionalDealerCard(dealer);
+	}
+
+	private void drawAdditionalPlayersCard(List<Player> players) {
+		for (Player player : players) {
+			drawAdditionalPlayerCard(player);
+		}
+	}
+
+	private void drawAdditionalPlayerCard(Player player) {
+		while (player.isDrawable() && isYes(player)) {
+			player.addCards(cardDeck.draw());
+			OutputView.printPlayerCard(player);
+		}
+	}
+
+	private boolean isYes(Player player) {
+		YesOrNo yesOrNo = YesOrNo.of(InputView.inputYesOrNo(player.getName()));
```

Comment:
바로 물어봐도 되지 않을까요? 
```suggestion
		return YesOrNo.isYes(InputView.inputYesOrNo(player.getName()));
```

YesOrNo보단... 내부 Value를 나타내는 네이밍 보단 어떤 타입인지 나타나내는 네이밍으로 바꿔주세요~

Reply:
직관적인 플레이어의 응답상태를 표현할 수 있도록 DrawResponse로 수정했습니다!